### ISSUE TITLE
[Request] Color for QCheckbox, QRadio and QToggle when unchecked

### ISSUE THREAD

**Issue by sebastienroul** [quasarframework/quasar#973]
Checkbox and toogle have a true / false value. If you set "color" attr, then component is colored when modele value is true.

My feature request is to let the border of the checkbox, and the background rectangle of the toggle colored when model value is false. In many apps, the color mean something (UX, functional, ...) so uncoloring the component remove this meaning and make it look like  "disabled" component.

It could be an new attribute off-color, (Boolean) that let/not let the color when model value is false.

I cant managed it for Toggle cause of using of the `currentColor` in CSS. But but adding the following ugly css does the job for checkbox : 
```
.q-option-inner.relative-position {
   color: inherit !important;
}
```
It could also been extends to the radio, ...

I cant find in the code where it is managed in the component checkbox, let me kow, I could do the PR.

Sample : https://photos.app.goo.gl/Y9L78RTVVQrKIQkq1
My request is only for the border of the checkbox - ** not for the label **

Regards

**Comment by rstoenescu** [comment]
Will do. Thanks for suggesting it.

**Comment by rstoenescu** [comment]
Will introduce a new prop (`unchecked-color`) instead.

Quasar is following [Material guidelines](https://material.io/guidelines/components/selection-controls.html#selection-controls-radio-button) which mention that the "unchecked" state should be a greyish color. But I agree with you, there are times when you can convey a message through color when "unchecked" too. So that's what this property will do.

**Comment by sebastienroul** [comment]
Thanks ! I am going to read  to understand where the code was hidden :)

**Comment by rstoenescu** [comment]
Ok, final design: added `keep-color` prop for QCheckbox, QRadio and QToggle, and also a bonus `dark` Boolean prop for displaying on darker backgrounds.

Will be available in v0.15.
